spec/helpers/source_control_helper_spec.rb

```diff
@@ -27,8 +27,10 @@
   config.configure_rspec_metadata!
   config.default_cassette_options = {
     match_requests_on: [:method,
-                        VCR.request_matchers.uri_without_param(:client_id, :client_secret, :access_token)]
+                        VCR.request_matchers.uri_without_param(:client_id, :client_secret, :access_token)],
```

Comment:
?

Reply:
You just delete the old VCR, and then is recorded again the next time you run specs.
If it is there, it will use it